Extensions/StyleCop/StyleCopTask/task.json

```diff
@@ -48,11 +48,21 @@
       "label": "Max. Violation Count",
       "defaultValue": "1000",
       "required": true,
-      "helpMarkDown": "Maximum violations before analysis stops",
+      "helpMarkDown": "Maximum project violations before analysis stops",
       "groupName":"advanced"
     }
     ,
     {
+      "name": "allowableViolationCount",
+      "type": "string",
+      "label": "Allowable Violation Count",
+      "defaultValue": "0",
+      "required": true,
+      "helpMarkDown": "Number of violations allowed before the task fails",
+      "groupName": "advanced"
+    }
+    ,
```

Comment:
Can you move this comma to after the brace on the previous line? Also do the same for the other lines so it's all in one commit if possible, I'm not sure why there is a mix of styles here.